force-app/main/default/profiles/Candidate access.profile-meta.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this used for? It is redundant? If not provide an explanation as to why it's used and write a description for the profile.

Reply:
Profile deleted. Complete